docs/sphinx/source/examples/evaluating-with-snowflake-arctic-embed.ipynb

```diff
@@ -0,0 +1,1774 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<picture>\n",
+    "  <source media=\"(prefers-color-scheme: dark)\" srcset=\"https://vespa.ai/assets/vespa-ai-logo-heather.svg\">\n",
+    "  <source media=\"(prefers-color-scheme: light)\" srcset=\"https://vespa.ai/assets/vespa-ai-logo-rock.svg\">\n",
+    "  <img alt=\"#Vespa\" width=\"200\" src=\"https://vespa.ai/assets/vespa-ai-logo-rock.svg\" style=\"margin-bottom: 25px;\">\n",
+    "</picture>\n",
+    "\n",
+    "# Evaluating retrieval with Snowflake arctic embed\n",
+    "\n",
+    "This notebook will demonstrate how different rank profiles in Vespa can be set up and evaluated.\n",
+    "For the rank profiles that use semantic search, we will use the small version of [Snowflake`s arctic embed model series](https://huggingface.co/Snowflake/snowflake-arctic-embed-s) for generating embeddings.\n",
```

Comment:
fix link parsing (maybe)
```suggestion
    "For the rank profiles that use semantic search, we will use the small version of [Snowflake's arctic embed model series](https://huggingface.co/Snowflake/snowflake-arctic-embed-s) for generating embeddings.\n",
```

Reply:
Sorry, still have some way to go with the Mac-keyboard :D